drivers/input/Kconfig

```diff
@@ -34,4 +34,9 @@ source "drivers/input/Kconfig.xpt2046"
 
 endmenu # Input Drivers
 
+config INPUT_MULTITOUCH_ENABLED
+		bool "Multitouch support enablement"
+		help
+			Enables multitouch support.
```

Comment:
this looks off in a different way now, the first two lines should have just a tab, the help text a tab and two spaces (yeah it's odd)